moveit_ros/planning_interface/CMakeLists.txt

```diff
@@ -1,28 +1,30 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(moveit_ros_planning_interface)
 
-add_compile_options(-std=c++14)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
 
 if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  moveit_msgs
-  moveit_ros_planning
-  moveit_ros_warehouse
-  moveit_ros_manipulation
-  moveit_ros_move_group
-  geometry_msgs
-  tf2
-  tf2_eigen
-  tf2_geometry_msgs
-  tf2_ros
-  roscpp
-  actionlib
-  rospy
-  rosconsole
-)
+find_package(ament_cmake REQUIRED)
+find_package(moveit_msgs REQUIRED)
+# find_package(moveit_ros_planning REQUIRED)
```

Comment:
Why is this commented right away? Is it because we're expecting to uncomment it once we port these submodules? If so, find below a suggestion:
```suggestion
# TODO: uncomment this as submodules become available
# find_package(moveit_ros_planning REQUIRED)
```

Reply:
Moveit_ros_planning depends from moveit_ros_perception, once I finish with that, ill uncomment it. I'll add the TODO: